client/src/main/java/tds/assessment/Segment.java

```diff
@@ -143,7 +168,7 @@ public void setFieldTestMinItems(int fieldTestMinItems) {
     /**
      * @return the maximum number of field test items in the {@link tds.assessment.Segment}
      */
-    public int getFieldTestMaxItems() {
+    public Integer getFieldTestMaxItems() {
```

Comment:
Do these have to be nullable?  For example, is there harm if this is zero if null in the db?